.github/workflows/e2e.yaml

```diff
@@ -257,7 +257,7 @@ jobs:
       matrix:
         distribution: ["k3s", "k8s"]
         test-suite-path: ${{fromJson(needs.get-testsuites-dir.outputs.matrix)}}
-        multinamespace-mode: ["false", "true"]
+        multinamespace-mode: ["false"]
```

Comment:
Oh btw if we get rid of this then completely delete that

Reply:
cleaned up 👍